src/components/Footer/style.css

```diff
@@ -0,0 +1,29 @@
+#h1{
+    font-style: "Quicksand";
+    font-size: 64px;
+    text-align: center;
+    margin-top: 0;
+    color: #000000;
+}
+
+#icon-box{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+}
+
+.icon{
+    margin-left: 2.3rem;
+    margin-right: 2.3rem;
```

Comment:
<img width="469" alt="Screen Shot 2022-01-27 at 3 14 14 PM" src="https://user-images.githubusercontent.com/33165426/151458905-666d5524-8f1e-4444-bd48-c977fdb46ac0.png">
Can we keep margin on the top and bottom also for when we overflow? just like a fixed amount on all sides